receiver/dcgmreceiver/client.go

```diff
@@ -100,12 +110,28 @@ func newClient(config *Config, logger *zap.Logger) (*dcgmClient, error) {
 	}, nil
 }
 
+// TODO: b/293585142 - Remove this function and use dcgm.Init() to check if DCGM is
+// installed after https://github.com/NVIDIA/go-dcgm/issues/13 is fixed
+func isDcgmInstalled() bool {
```

Comment:
I'd add a comment explaining what this function does, e.g.:
```
// isDcgmInstalled returns true if it was able to load the dcgm shared library.
// TODO(…): …
func isDcgmInstalled() bool {
	…
}
```

Reply:
Done.